lib/componentRelay.ts

```diff
@@ -224,20 +228,33 @@ export default class ComponentRelay {
     Logger.info('Waiting for messages...')
   }
 
-  private registerKeyDownEventListener() {
+  private registerKeyboardEventListeners() {
     this.keyDownEventListener = (event: KeyboardEvent) => {
       Logger.info(`A key has been pressed: ${event.key}`)
 
       if (event.ctrlKey) {
-        this.keyPressed(KeyboardModifier.Ctrl)
+        this.keyDownEvent(KeyboardModifier.Ctrl)
       } else if (event.shiftKey) {
-        this.keyPressed(KeyboardModifier.Shift)
+        this.keyDownEvent(KeyboardModifier.Shift)
       } else if (event.metaKey || event.key === 'Meta') {
-        this.keyPressed(KeyboardModifier.Meta)
+        this.keyDownEvent(KeyboardModifier.Meta)
```

Comment:
Should we check for event.key as a fallback for Ctrl and Shift as well?

Reply:
We could, but [this page](https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/ctrlKey) doesn't mention any unexpected behaviors like on [metaKey property](https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/metaKey)